cmd/fluxd/main.go

```diff
@@ -114,6 +114,7 @@ func main() {
 		token       = fs.String("token", "", "Authentication token for upstream service")
 
 		dockerConfig = fs.String("docker-config", "", "path to a docker config to use for image registry credentials")
+		k8sNamespaceWhitelist = fs.StringSlice("k8s-namespace-whitelist", []string{}, "Optional, comma separated list of namespaces to monitor for workloads")
```

Comment:
Let's mark this as experimental, for the minute. How about, `"Experimental, optional: restrict the view of the cluster to the namespaces listed. All namespaces are included if this is not set."`

Reply:
Remember to also update the site/daemon.md table with this too.